CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+---
+
+Thank you for your interest in contributing to the P4 Compiler tutorials repository! Your contributions are important and will help to improve the project for everyone. Before you begin, please consider the guidelines below.
+
+## Getting Started
+
+- Make sure you have [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed on your machine.
+- Fork the repository and clone it locally.
+  
+  ```
+  git clone https://github.com/p4lang/tutorials.git
+  ```
+  
+- Create a new branch for your contributions
+  
+  ```
+  git checkout -b feature-branch-name
+  ```
+
+## Making Changes
+
+- Before making changes, ensure that you're working on the latest version of the `master` branch
+  
+  ```
+  git pull origin master
+  ```
+- Make your changes following the project's [Coding conventions](https://github.com/p4lang/p4c/tree/main/docs#coding-conventions) and [guidelines](https://github.com/p4lang/p4c/blob/main/docs/CodingStandardPhilosophy.md).
+
+## Committing Changes
+
+- Stage your changes:
+  
+  ```
+  git add file-name(s)
+  ```
+- Commit your changes with a descriptive message:
+  
+  ```
+  git commit -m "Add feature"
+  ```
+- Push your changes to your forked repository:
+  
+  ```
+  git push origin feature-branch-name
+  ```
+
+## Submitting a Pull Request
+
+- Navigate to your forked repository on GitHub.
+- Create a new pull request (PR) targeting the `master` branch of the p4 tutorial repository.
+- Provide a clear title and description for your PR, explaining the purpose of your changes.
+- Ensure that your PR follows our code [guidelines](https://github.com/p4lang/p4c/blob/main/docs/CodingStandardPhilosophy.md).
```

Comment:
Doesn't Github have instructions we could reference for creating a fork, cloning it, creating a branch, creating a PR, etc.?  There is nothing unique to these steps about the tutorials repo, as far as I am aware.